packages/components/addon/components/hds/tabs/panel.js

```diff
@@ -0,0 +1,56 @@
+import Component from '@glimmer/component';
+import { cached } from '@glimmer/tracking';
+import { guidFor } from '@ember/object/internals';
+import { action } from '@ember/object';
+
+export default class HdsTabsIndexComponent extends Component {
+  /**
+   * Generates a unique ID for the Panel
+   *
+   * @param panelId
+   */
+  // TODO: switch to {{unique-id}} when we're able
+  // (https://github.com/emberjs/ember.js/blob/089a021b1b5c5f8ea1cb574fcd841a73af7b2031/packages/%40ember/-internals/glimmer/lib/helpers/unique-id.ts#L6)
```

Comment:
Ok this was a comment from one of the reviews I added. I'll commit your deletion, thanks.

Reply:
Unsure if the "can" here is related to the functionality or this feature actually existing, but FWIW we are on a version of ember that does include this now and there is also another [polyfill](https://www.npmjs.com/package/ember-unique-id-helper-polyfill) to backport this for all our consumers who are not on the later versions of ember.

Reply:
@Dhaulagiri with _"we can't use here"_ I meant in JS code (unless I am missing something: looking at the RFC to me seems it can be used only in templates, not in backing classes).
In the specific case of the tabs there are two things that makes me think we can't use it as is:
- we manipulate the `unique-id` (we add a prefix to it)
- we need it on the JS side of things: how would you access the value of `{{unique-id}}` in JS? I can't find references about it.

Reply:
🙇  thanks for the explanation. seems prudent to ignore me here 😅